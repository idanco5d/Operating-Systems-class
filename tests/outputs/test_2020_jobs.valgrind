==20658== Memcheck, a memory error detector
==20658== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==20658== Using Valgrind-3.20.0 and LibVEX; rerun with -h for copyright info
==20658== Command: /mnt/c/Users/illus/os/hw/1/smash
==20658== Parent PID: 20622
==20658== 
==20658== 
==20658== FILE DESCRIPTORS: 4 open (3 std) at exit.
==20658== Open file descriptor 3: /mnt/c/Users/illus/os/hw/1/tests/outputs/test_2020_jobs.valgrind
==20658==    <inherited from parent>
==20658== 
==20658== 
==20658== HEAP SUMMARY:
==20658==     in use at exit: 0 bytes in 0 blocks
==20658==   total heap usage: 967 allocs, 967 frees, 122,790 bytes allocated
==20658== 
==20658== All heap blocks were freed -- no leaks are possible
==20658== 
==20658== For lists of detected and suppressed errors, rerun with: -s
==20658== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
