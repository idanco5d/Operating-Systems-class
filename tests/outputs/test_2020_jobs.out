smash> smash> smash> smash> smash> smash> smash> smash> [1] sleep 100& : 8832 0 secs
[2] sleep 200& : 8834 0 secs
[3] sleep 300& : 8842 0 secs
[4] sleep 400& : 8853 0 secs
[5] sleep 500& : 8857 0 secs
[6] sleep 600& : 8858 0 secs
[7] sleep 700& : 8861 0 secs
smash> smash> smash> [1] sleep 100& : 8832 0 secs
[2] sleep 200& : 8834 0 secs
[3] sleep 300& : 8842 0 secs
[4] sleep 400& : 8853 0 secs
[5] sleep 500& : 8857 0 secs
[6] sleep 600& : 8858 0 secs
[7] sleep 700& : 8861 0 secs
[8] sleep 800& : 8865 0 secs
[9] sleep 900& : 8867 0 secs
smash> signal number 9 was sent to pid 8842
smash> [1] sleep 100& : 8832 1 secs
[2] sleep 200& : 8834 1 secs
[4] sleep 400& : 8853 1 secs
[5] sleep 500& : 8857 1 secs
[6] sleep 600& : 8858 1 secs
[7] sleep 700& : 8861 1 secs
[8] sleep 800& : 8865 1 secs
[9] sleep 900& : 8867 1 secs
smash> smash> [1] sleep 100& : 8832 1 secs
[2] sleep 200& : 8834 1 secs
[4] sleep 400& : 8853 1 secs
[5] sleep 500& : 8857 1 secs
[6] sleep 600& : 8858 1 secs
[7] sleep 700& : 8861 1 secs
[8] sleep 800& : 8865 1 secs
[9] sleep 900& : 8867 1 secs
[10] sleep 1000& : 9006 0 secs
smash> signal number 9 was sent to pid 9006
smash> [1] sleep 100& : 8832 2 secs
[2] sleep 200& : 8834 2 secs
[4] sleep 400& : 8853 2 secs
[5] sleep 500& : 8857 2 secs
[6] sleep 600& : 8858 2 secs
[7] sleep 700& : 8861 2 secs
[8] sleep 800& : 8865 2 secs
[9] sleep 900& : 8867 2 secs
smash> smash> [1] sleep 100& : 8832 2 secs
[2] sleep 200& : 8834 2 secs
[4] sleep 400& : 8853 2 secs
[5] sleep 500& : 8857 2 secs
[6] sleep 600& : 8858 2 secs
[7] sleep 700& : 8861 2 secs
[8] sleep 800& : 8865 2 secs
[9] sleep 900& : 8867 2 secs
[10] sleep 1001& : 9096 0 secs
smash> sleep 1001& : 9096
smash: got ctrl-Z
smash: process 9096 was stopped
smash> [1] sleep 100& : 8832 2 secs
[2] sleep 200& : 8834 2 secs
[4] sleep 400& : 8853 2 secs
[5] sleep 500& : 8857 2 secs
[6] sleep 600& : 8858 2 secs
[7] sleep 700& : 8861 2 secs
[8] sleep 800& : 8865 2 secs
[9] sleep 900& : 8867 2 secs
[10] sleep 1001& : 9096 0 secs (stopped)
smash> sleep 1001& : 9096
smash: got ctrl-Z
smash: process 9096 was stopped
smash> [1] sleep 100& : 8832 3 secs
[2] sleep 200& : 8834 3 secs
[4] sleep 400& : 8853 3 secs
[5] sleep 500& : 8857 3 secs
[6] sleep 600& : 8858 3 secs
[7] sleep 700& : 8861 3 secs
[8] sleep 800& : 8865 3 secs
[9] sleep 900& : 8867 3 secs
[10] sleep 1001& : 9096 1 secs (stopped)
smash> sleep 1001& : 9096
smash: got ctrl-Z
smash: process 9096 was stopped
smash> [1] sleep 100& : 8832 3 secs
[2] sleep 200& : 8834 3 secs
[4] sleep 400& : 8853 3 secs
[5] sleep 500& : 8857 3 secs
[6] sleep 600& : 8858 3 secs
[7] sleep 700& : 8861 3 secs
[8] sleep 800& : 8865 3 secs
[9] sleep 900& : 8867 3 secs
[10] sleep 1001& : 9096 1 secs (stopped)
smash> sleep 1001& : 9096
smash: got ctrl-Z
smash: process 9096 was stopped
smash> [1] sleep 100& : 8832 4 secs
[2] sleep 200& : 8834 4 secs
[4] sleep 400& : 8853 4 secs
[5] sleep 500& : 8857 4 secs
[6] sleep 600& : 8858 4 secs
[7] sleep 700& : 8861 4 secs
[8] sleep 800& : 8865 4 secs
[9] sleep 900& : 8867 4 secs
[10] sleep 1001& : 9096 2 secs (stopped)
smash> smash: got ctrl-C
[1] sleep 100& : 8832 4 secs
[2] sleep 200& : 8834 4 secs
[4] sleep 400& : 8853 4 secs
[5] sleep 500& : 8857 4 secs
[6] sleep 600& : 8858 4 secs
[7] sleep 700& : 8861 4 secs
[8] sleep 800& : 8865 4 secs
[9] sleep 900& : 8867 4 secs
[10] sleep 1001& : 9096 2 secs (stopped)
smash> smash: got ctrl-Z
smash: process 9247 was stopped
smash> [1] sleep 100& : 8832 8 secs
[2] sleep 200& : 8834 8 secs
[4] sleep 400& : 8853 8 secs
[5] sleep 500& : 8857 8 secs
[6] sleep 600& : 8858 8 secs
[7] sleep 700& : 8861 8 secs
[8] sleep 800& : 8865 8 secs
[9] sleep 900& : 8867 8 secs
[10] sleep 1001& : 9096 6 secs (stopped)
[11] sleep 1002 : 9247 0 secs (stopped)
smash> sleep 1002 : 0
smash> [1] sleep 100& : 8832 8 secs
[2] sleep 200& : 8834 8 secs
[4] sleep 400& : 8853 8 secs
[5] sleep 500& : 8857 8 secs
[6] sleep 600& : 8858 8 secs
[7] sleep 700& : 8861 8 secs
[8] sleep 800& : 8865 8 secs
[9] sleep 900& : 8867 8 secs
[10] sleep 1001& : 9096 6 secs (stopped)
[11] sleep 1002 : 9247 0 secs
smash> smash: got ctrl-C
smash: process 9445 was killed
smash> smash> smash: got ctrl-C
smash: got ctrl-C
smash: got ctrl-Z
smash: got ctrl-Z
[1] sleep 100& : 8832 2e+01 secs
[2] sleep 200& : 8834 2e+01 secs
[4] sleep 400& : 8853 2e+01 secs
[5] sleep 500& : 8857 2e+01 secs
[6] sleep 600& : 8858 2e+01 secs
[7] sleep 700& : 8861 2e+01 secs
[8] sleep 800& : 8865 2e+01 secs
[9] sleep 900& : 8867 2e+01 secs
[10] sleep 1001& : 9096 1e+01 secs (stopped)
[11] sleep 1002 : 9247 7 secs
smash> smash: got ctrl-Z
smash: process 9896 was stopped
smash> smash: got ctrl-Z
smash: process 9919 was stopped
smash> smash: got ctrl-Z
smash: process 9944 was stopped
smash> sleep 1100 : 9896
smash: got ctrl-Z
smash: process 9896 was stopped
smash> sleep 1100 : 9896
smash: got ctrl-Z
smash: process 9896 was stopped
smash> sleep 1100 : 9896
smash: got ctrl-Z
smash: process 9896 was stopped
smash> [1] sleep 100& : 8832 2e+01 secs
[2] sleep 200& : 8834 2e+01 secs
[4] sleep 400& : 8853 2e+01 secs
[5] sleep 500& : 8857 2e+01 secs
[6] sleep 600& : 8858 2e+01 secs
[7] sleep 700& : 8861 2e+01 secs
[8] sleep 800& : 8865 2e+01 secs
[9] sleep 900& : 8867 2e+01 secs
[10] sleep 1001& : 9096 2e+01 secs (stopped)
[11] sleep 1002 : 9247 1e+01 secs
[12] sleep 1100 : 9896 3 secs (stopped)
[13] sleep 1200 : 9919 2 secs (stopped)
[14] sleep 1200 : 9944 2 secs (stopped)
smash> sleep 1002 : 9247
smash: got ctrl-Z
smash: process 9247 was stopped
smash> sleep 1002 : 9247
smash: got ctrl-C
smash: process 9247 was killed
smash> smash> smash> smash> [1] sleep 100& : 8832 2e+01 secs
[2] sleep 200& : 8834 2e+01 secs
[4] sleep 400& : 8853 2e+01 secs
[5] sleep 500& : 8857 2e+01 secs
[6] sleep 600& : 8858 2e+01 secs
[7] sleep 700& : 8861 2e+01 secs
[8] sleep 800& : 8865 2e+01 secs
[9] sleep 900& : 8867 2e+01 secs
[10] sleep 1001& : 9096 2e+01 secs (stopped)
[12] sleep 1100 : 9896 4 secs (stopped)
[13] sleep 1200 : 9919 3 secs (stopped)
[14] sleep 1200 : 9944 3 secs (stopped)
smash> sleep 1100 : 4
smash> smash> smash> [1] sleep 100& : 8832 2e+01 secs
[2] sleep 200& : 8834 2e+01 secs
[4] sleep 400& : 8853 2e+01 secs
[5] sleep 500& : 8857 2e+01 secs
[6] sleep 600& : 8858 2e+01 secs
[7] sleep 700& : 8861 2e+01 secs
[8] sleep 800& : 8865 2e+01 secs
[9] sleep 900& : 8867 2e+01 secs
[10] sleep 1001& : 9096 2e+01 secs (stopped)
[12] sleep 1100 : 9896 4 secs
[13] sleep 1200 : 9919 3 secs (stopped)
[14] sleep 1200 : 9944 3 secs (stopped)
smash> smash> smash> [1] sleep 100& : 8832 2e+01 secs
[2] sleep 200& : 8834 2e+01 secs
[4] sleep 400& : 8853 2e+01 secs
[5] sleep 500& : 8857 2e+01 secs
[6] sleep 600& : 8858 2e+01 secs
[7] sleep 700& : 8861 2e+01 secs
[8] sleep 800& : 8865 2e+01 secs
[9] sleep 900& : 8867 2e+01 secs
[10] sleep 1001& : 9096 2e+01 secs (stopped)
[12] sleep 1100 : 9896 4 secs
[13] sleep 1200 : 9919 3 secs (stopped)
[14] sleep 1200 : 9944 3 secs (stopped)
smash> signal number 9 was sent to pid 8832
smash> smash> [2] sleep 200& : 8834 2e+01 secs
[4] sleep 400& : 8853 2e+01 secs
[5] sleep 500& : 8857 2e+01 secs
[6] sleep 600& : 8858 2e+01 secs
[7] sleep 700& : 8861 2e+01 secs
[8] sleep 800& : 8865 2e+01 secs
[9] sleep 900& : 8867 2e+01 secs
[10] sleep 1001& : 9096 2e+01 secs (stopped)
[12] sleep 1100 : 9896 7 secs
[13] sleep 1200 : 9919 6 secs (stopped)
[14] sleep 1200 : 9944 6 secs (stopped)
smash> signal number 9 was sent to pid 8834
smash> signal number 9 was sent to pid 8853
smash> smash> smash> smash> [5] sleep 500& : 8857 2e+01 secs
[6] sleep 600& : 8858 2e+01 secs
[7] sleep 700& : 8861 2e+01 secs
[8] sleep 800& : 8865 2e+01 secs
[9] sleep 900& : 8867 2e+01 secs
[10] sleep 1001& : 9096 2e+01 secs (stopped)
[12] sleep 1100 : 9896 9 secs
[13] sleep 1200 : 9919 8 secs (stopped)
[14] sleep 1200 : 9944 8 secs (stopped)
smash> smash> [5] sleep 500& : 8857 2e+01 secs
[6] sleep 600& : 8858 2e+01 secs
[7] sleep 700& : 8861 2e+01 secs
[8] sleep 800& : 8865 2e+01 secs
[9] sleep 900& : 8867 2e+01 secs
[10] sleep 1001& : 9096 2e+01 secs (stopped)
[12] sleep 1100 : 9896 9 secs
[13] sleep 1200 : 9919 8 secs (stopped)
[14] sleep 1200 : 9944 8 secs (stopped)
[15] sleep 3& : 10378 0 secs
smash> [5] sleep 500& : 8857 3e+01 secs
[6] sleep 600& : 8858 3e+01 secs
[7] sleep 700& : 8861 3e+01 secs
[8] sleep 800& : 8865 3e+01 secs
[9] sleep 900& : 8867 3e+01 secs
[10] sleep 1001& : 9096 3e+01 secs (stopped)
[12] sleep 1100 : 9896 1e+01 secs
[13] sleep 1200 : 9919 1e+01 secs (stopped)
[14] sleep 1200 : 9944 1e+01 secs (stopped)
smash> [5] sleep 500& : 8857 3e+01 secs
[6] sleep 600& : 8858 3e+01 secs
[7] sleep 700& : 8861 3e+01 secs
[8] sleep 800& : 8865 3e+01 secs
[9] sleep 900& : 8867 3e+01 secs
[10] sleep 1001& : 9096 3e+01 secs (stopped)
[12] sleep 1100 : 9896 1e+01 secs
[13] sleep 1200 : 9919 1e+01 secs (stopped)
[14] sleep 1200 : 9944 1e+01 secs (stopped)
smash> [5] sleep 500& : 8857 3e+01 secs
[6] sleep 600& : 8858 3e+01 secs
[7] sleep 700& : 8861 3e+01 secs
[8] sleep 800& : 8865 3e+01 secs
[9] sleep 900& : 8867 3e+01 secs
[10] sleep 1001& : 9096 3e+01 secs (stopped)
[12] sleep 1100 : 9896 1e+01 secs
[13] sleep 1200 : 9919 1e+01 secs (stopped)
[14] sleep 1200 : 9944 1e+01 secs (stopped)
smash> [5] sleep 500& : 8857 3e+01 secs
[6] sleep 600& : 8858 3e+01 secs
[7] sleep 700& : 8861 3e+01 secs
[8] sleep 800& : 8865 3e+01 secs
[9] sleep 900& : 8867 3e+01 secs
[10] sleep 1001& : 9096 3e+01 secs (stopped)
[12] sleep 1100 : 9896 2e+01 secs
[13] sleep 1200 : 9919 1e+01 secs (stopped)
[14] sleep 1200 : 9944 1e+01 secs (stopped)
smash> sleep 1200 : 9944
